vowpalwabbit/cb_explore.cc

```diff
@@ -302,6 +302,11 @@ base_learner* cb_explore_setup(options_i& options, vw& all)
     ss << data->cbcs.num_actions;
     options.insert("cb", ss.str());
   }
+  
+  if (data->epsilon < 0.0 || data->epsilon > 1.0)
+  {
+    THROW("cb_explore_epsilon must be in [0,1]");
```

Comment:
The use may not know what `cb_explore_epsilon` is, it doesn't correspond to an option they inputted. Can you make the error message clearer with that in mind?

Reply:
Yes, I have modified the error messages. Please review them, thanks!